src/DependencyInjection/SonataNotificationExtension.php

```diff
@@ -163,19 +161,14 @@ public function configureBackends(ContainerBuilder $container, $config)
     }
 
     /**
-     * @param array $config
+     * NEXT_MAJOR: remove this method, since all Doctrine mappings are already
+     *             described in Resources/config/doctrine/BaseMessage.orm.xml.
+     *
+     * @deprecated Since version 3.x, to be removed in 4.0.
+     *             EasyExtends should be used only for associations between entities.
```

Comment:
should we throw a deprecation also?

Reply:
This method was useless before (indexes were added to `BaseMessage.orm.xml` on april 2012, see https://github.com/sonata-project/SonataNotificationBundle/commit/b8a8b608fdb8cca9b04f1e854f90508964df21c1), and it will be useless now (because it will be empty =). So, i thougth, throwing exception is not necessary.

Reply:
he said deprecation (not exception) :-)

Reply:
@OskarStark do you mean `trigger_error('..', E_USER_DEPRECATED)`?

Reply:
yes

Reply:
@kunicmarko20 @OskarStark you are right, i'll add deprecation =) !